plugin/al/CMakeLists.txt

```diff
@@ -51,6 +51,27 @@ set(AL_USDMAYA_LOCATION_NAME
     "Name of the environment variable used to store AL_USDMaya installation location"
 )
 
+#------------------------------------------------------------------------------
+# compiler configuration
+#------------------------------------------------------------------------------
+# CXXDefaults will set a variety of variables for the project.
+# Consume them here. This is an effort to keep the most common
+# build files readable.
+list(APPEND CMAKE_MODULE_PATH
+    ${PROJECT_SOURCE_DIR}/../pxr/cmake/defaults
```

Comment:
Related to the comment above about setting the C++ language standard in `mayaUsd_compile_config()`, wouldn't we *also* need the change in #395 so that `CXXDefaults` specifies the language standard here? This does *not* use `mayaUsd_compile_config()`, so it seems like no standard is being specified.

Reply:
I originally didn't want to bring mayaUsd_compile_config() into either AL or Pixar since that is outside the scope of this PR.

Both plugin/pixar and plugins/al targets are not affected by mayaUsd_compile_config intentionally and they are using the same compiler configuration as before.

You are correct, I didn't bring #395 but I believe I should have. :) I will make the change with proper switch between C++11 and C++14.

Reply:
Please see dc898ce4b2